Jenkinsfile

```diff
@@ -147,7 +147,8 @@ sed -e 's#{BUILD_BRANCH}#${env.BRANCH_NAME}#g' \
       '.pax/zowe-smpe.zip',
       '.pax/smpe-promote.tar',
       '.pax/pd.htm',
-      '.pax/smpe-build-logs.pax.Z'
+      '.pax/smpe-build-logs.pax.Z',
+      '.pax/AZWE001.TMP0001.zip'
```

Comment:
We are missing .pax/AZWE001.zip. As soon as Nayer pushes the ptf-bucket into the staging branch, zowe-smpe.zip will no longer hold the FMID but the PTF. (Note, zowe-smpe.zip is intended to hold what should be published on zowe.org.)  Both FMID and PTF should always be available in Artifactory for automated testing. (Also, the FMID is needed to simplify loading a PTF build in Shopz.)